scripts/SANS/ISISCommandInterface.py

```diff
@@ -44,6 +44,7 @@ def appwidgets():
 
 def SetVerboseMode(state):
     # TODO: this needs to be on the reducer
+    global _VERBOSE_
```

Comment:
But now the function actually works. Since this previously did nothing but is called by parts of Mantid I might as well replace it with `pass`, to make it obvious this function does nothing?